manual/src/refman/exten.etex

```diff
@@ -123,5 +123,30 @@ that are implemented in OCaml, but not described in chapter \ref{c:refman}.
 %HEVEA\cutname{effects.html}
 \input{effects.tex}
 
+\section{s:memory-model}{Memory model for parallel OCaml programs}
+%HEVEA\cutname{lang_memorymodel.html}
+(Introduced in 5.0)
+
+OCaml 5.0 has introduced support for running parallel OCaml code in the same
+process.
+Once an OCaml program has more than one \stdmoduleref{Domain} running the
+following semantic changes take effect:
+\begin{description}
+
+  \item[DRF-SC guarantee] is a consistent semantics for data race free parts of the program
```

Comment:
DRF-SC holds even for sequential programs trivially since there are no data races. I wouldn't classify this as a semantic change that takes effect when the OCaml program has more than one domain.